css/decanter.css

```diff
@@ -1,9 +1,3 @@
-html {
```

Comment:
Is this no longer a thing? Something to do with normalize?

Reply:
Yep, this is from Normalize, which we're still using. That said, I'd say that it's still a thing.

https://necolas.github.io/normalize.css/8.0.0/normalize.css